CMakeLists.txt

```diff
@@ -166,6 +167,35 @@ if(NOT mxnet_LINKER_LIBS)
   set(mxnet_LINKER_LIBS "")
 endif(NOT mxnet_LINKER_LIBS)
 
+if(USE_NGRAPH AND NOT MSVC)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-comment")
+  set(NGRAPH_INSTALL_PREFIX ${CMAKE_BINARY_DIR}/3rdparty/ngraph-mxnet-bridge)
+
+  if(NOT MSVC)
+    set(ARCH_OPT_FLAGS "-mtune=generic")
+  endif()
+  add_subdirectory(3rdparty/ngraph-mxnet-bridge)
+
+  include_directories(3rdparty/ngraph-mxnet-bridge/src)
+  include_directories(${CMAKE_BINARY_DIR}/3rdparty/ngraph-mxnet-bridge/include)
+  
+  add_definitions(-DMXNET_USE_NGRAPH=1)
+  
+  list(APPEND mxnet_LINKER_LIBS ${NGRAPH_INSTALL_PREFIX}/lib/libngraph.so)
+  list(APPEND mxnet_LINKER_LIBS ${NGRAPH_INSTALL_PREFIX}/lib/libmkldnn.so)
+  list(APPEND mxnet_LINKER_LIBS ${NGRAPH_INSTALL_PREFIX}/lib/libiomp5.so)
+  list(APPEND mxnet_LINKER_LIBS ${NGRAPH_INSTALL_PREFIX}/lib/libmklml_intel.so)
+  list(APPEND mxnet_LINKER_LIBS ${NGRAPH_INSTALL_PREFIX}/lib/libcpu_backend.so)
+  
+  if(USE_NGRAPH_DISTRIBUTED)
```

Comment:
I thought we are not including distributed stuff in the first PR?

Reply:
Good catch